requirements/default.txt

```diff
@@ -169,6 +169,9 @@ elasticsearch==1.2.0
 # sha256: 08GfJqajRinBjHdfWd_F3VlXZMcitXotpW6_tpuU5Ec
 enum34==1.0.4
 
+# sha256: 1wEkmeUt5aRBPiKu1R34lIUzaFRS3-FrQQAfKPnOWxw
+factory-boy==2.4.1
```

Comment:
Doh! I started to, but then things got hard and I stopped, because it was taking too much time.